profiles/models.py

```diff
@@ -102,3 +106,29 @@ class DivisionOfInterest(models.Model):
         on_delete=models.CASCADE,
         related_name="division_of_interest",
     )
+
+
+class Contact(models.Model):
+    profile = models.ForeignKey(Profile, on_delete=models.CASCADE)
+    primary = models.BooleanField(default=False)
+
+    class Meta:
+        abstract = True
+
+
+class Phone(Contact):
+    phone = models.CharField(max_length=255, null=True, blank=False)
+    phone_type = models.CharField(max_length=32, choices=PHONE_TYPES, blank=False)
+
+
+class Email(Contact):
+    email = models.EmailField(null=True, blank=True)
+    email_type = models.CharField(max_length=32, choices=EMAIL_TYPES, blank=False)
+
+
+class Address(Contact):
+    address = models.CharField(max_length=128, blank=False)
+    postal_code = models.CharField(max_length=5, blank=False)
+    city = models.CharField(max_length=64, blank=False)
+    country = models.CharField(max_length=64, blank=True)
```

Comment:
Just occurred to me that we could store the country as a 2 character iso code. Perhaps even with this: https://pypi.org/project/django-countries/